go.mod

```diff
@@ -120,3 +120,5 @@ require (
 )
 
 go 1.18
+
+replace github.com/codeready-toolchain/toolchain-common => github.com/mfrancisc/toolchain-common v0.0.0-20230105093252-b18c1ea3a028
```

Comment:
reminder ;)

Reply:
changed in [2c13a2f](https://github.com/codeready-toolchain/toolchain-e2e/pull/645/commits/2c13a2fa53fac325af469c3435cf09662f097ce4)